test/test_cpu/test_model_scope.py

```diff
@@ -25,10 +26,18 @@ def setUpClass(self):
         self.saved_path = "./saved"
         self.dataset = LLMDataLoader()
 
+        self.source_path, self.cache_path = "/tf_dataset/auto_round/modelscope", "/home/hostuser/.cache/modelscope"
+        if os.path.exists(self.source_path):
+            if not os.path.exists("/home/hostuser/.cache"):
+                os.makedirs("/home/hostuser/.cache")
+            shutil.copytree(self.source_path, self.cache_path, dirs_exist_ok=True)
```

Comment:
Instead of copying model, we can use `os.environ["MODELSCOPE_CACHE"]="/tf_dataset/auto_round/modelscope"`